libs/getClient.js

```diff
@@ -154,6 +154,7 @@ module.exports = (tokenPath, cozyUrl, docTypes) => {
         )
       } else {
         console.warn(err)
+        throw new Error('Could not create client, see error above.')
```

Comment:
I find it good to be able to see the error that we swallow. If you know a better way to link the Error thrown with the original error, I am all ears.

Reply:
```
class ChainedError extends Error { 
  constructor(message, parent) {
    this.parent = parent
    super(message)
  }
}
throw new ChainedError('cound not create client, see error above.', err)
``` 

Would it match your need ?

Reply:
Thanks, I had to experiment it so that throwing the error directly prints the right message/stack.

```
class ChainedError extends Error { 
  constructor(message, original) {
    super(message)
    this.stack = original.stack + '\n' + this.stack
  }
}

const err = new Error('Parent error')
const err2 = new ChainedError('could not create client, see error above.', err)
throw new ChainedError('Deep', err2)
```

Reply:
See https://github.com/cozy/ACH/pull/48/commits/8063437428bf68756b7676ae6727eb1d9075882f